app/views/submissions/_assigned_evaluators.html.erb

```diff
@@ -48,20 +48,22 @@
                     <span class="margin-x-1"><%= evaluator_score(assignment).formatted_score %></span>
                   </td>
                   <td data-label="">
-                    <div class="display-flex flex-wrap">
-                      <% if assignment&.evaluation&.completed_at %>
-                        <%= link_to(evaluation_path(assignment.evaluation)) do %>
-                          <button class="usa-button font-body-3xs text-no-wrap padding-x-1 margin-bottom-1">
-                            View Evaluation
-                          </button>
+                    <% unless @submission.winner? %>
+                      <div class="display-flex flex-wrap">
+                        <% if assignment&.evaluation&.completed_at %>
+                          <%= link_to(edit_evaluation_path(assignment.evaluation)) do %>
```

Comment:
I think it's okay to leave out the edit evaluation parts and just disable assign/unassignment from submission winners

Reply:
Sounds good!

Reply:
also the evaluation is still visible after being selected as a winner, but that view would be read-only, which Chris will also add to the view. basically the "View Evaluation" button should always be available afaik.